appbars/src/main/java/com/decathlon/vitamin/compose/appbars/bottomnavigations/VitaminBottomNavigations.kt

```diff
@@ -31,19 +64,207 @@ object VitaminBottomNavigations {
     ) {
         BottomNavigation(
             modifier = modifier,
-            backgroundColor = colors.background,
-            contentColor = colors.unSelected
+            backgroundColor = colors.backgroundColor,
+            contentColor = colors.unSelectedColor
         ) {
             actions.forEach {
-                BottomNavigationItem(
+                VitaminBottomNavigationItem(
                     selected = it.selected,
                     onClick = { it.onClick() },
                     icon = { Icon(painter = it.icon, contentDescription = it.contentDescription) },
                     label = it.content,
-                    selectedContentColor = colors.selected,
-                    unselectedContentColor = colors.unSelected
+                    colors = colors
                 )
             }
         }
     }
+
+    @Composable
+    internal fun RowScope.VitaminBottomNavigationItem(
+        selected: Boolean,
+        onClick: () -> Unit,
+        icon: @Composable () -> Unit,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        alwaysShowLabel: Boolean = true,
+        colors: BottomNavigationColors = VitaminBottomNavigationColors.primary(),
+        interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+        label: @Composable (() -> Unit)? = null
+    ) {
+        val styledLabel: @Composable (() -> Unit)? = label?.let {
+            @Composable {
+                val style = MaterialTheme.typography.caption.copy(textAlign = TextAlign.Center)
+                ProvideTextStyle(style, content = label)
+            }
+        }
+        val ripple = rememberRipple(bounded = false, color = colors.selectedContentColor)
+
+        Box(
+            modifier
+                .selectable(
+                    selected = selected,
+                    onClick = onClick,
+                    enabled = enabled,
+                    role = Role.Tab,
+                    interactionSource = interactionSource,
+                    indication = ripple
+                )
+                .weight(1f),
+            contentAlignment = Alignment.Center
+        ) {
+            VitaminBottomNavigationItemBaselineLayout(
+                icon = icon,
+                label = styledLabel,
+                colors = colors,
+                selected = selected,
+                alwaysShowLabel = alwaysShowLabel
+            )
+        }
+    }
+
+    @Composable
+    private fun VitaminBottomNavigationItemBaselineLayout(
+        selected: Boolean,
+        alwaysShowLabel: Boolean = true,
+        icon: @Composable () -> Unit,
+        colors: BottomNavigationColors = VitaminBottomNavigationColors.primary(),
+        label: @Composable (() -> Unit)?
```

Comment:
Label in your VitaminBottomNavigationItem has default value = null. Why here there is no default value ?